### PR TITLE
Hotfix/tests

### PR DESCRIPTION
**Pull request by Apehaenger**
Fixed some tiny issues I had when trying to run the tests.

### CODE REVIEW

File: test/integration/server/bootstrap.js [hunk 1]
```diff
@@ -58,7 +58,10 @@ let inserts = [calendarData, contactsData].map(tableData => {
 ].concat(inserts).forEach(command => {
   debug(`exec: ${command}`);
   setup(function(done) {
-    exec(command, { cwd: __dirname + '/SabreDAV' }, () => done());
+    exec(command, { cwd: __dirname + '/SabreDAV' }, (error) => {
```
[the remark this answers is not in the document]
Comment (Apehaenger) [reply test/integration/server/bootstrap.js]: Cause a failing command would not abort the tests.
In my case I did not had the sqlite3 command in the normal path. Took me some hours till I recognized that the preload of the DB doesn't happen.

[the remark this answers is not in the document]
Comment (lambdabaa) [reply test/integration/server/bootstrap.js]: Right that makes sense to me when it's `() => done()` like it was before but just `done` should pass along the error if it's defined, right?

[the remark this answers is not in the document]
Comment (Apehaenger) [reply test/integration/server/bootstrap.js]: Wait... now I understand. My fault!
You're right, just tested it. Your correction is right (and for sure better) ;-) !

**Comment by lambdabaa** [comment]
Great, thanks!

**Pull request merged**